src/sentry/workflow_engine/handlers/condition/slow_condition_query_handlers.py

```diff
@@ -180,45 +169,78 @@ def get_rate_bulk(
         return result
 
 
-class BaseEventFrequencyCountHandler:
-    comparison_json_schema = {
-        "type": "object",
-        "properties": {
-            "interval": {"type": "string", "enum": list(STANDARD_INTERVALS.keys())},
-            "value": {"type": "integer", "minimum": 0},
-        },
-        "required": ["interval", "value"],
-        "additionalProperties": False,
-    }
-
-    @staticmethod
-    def evaluate_value(value: WorkflowJob, comparison: Any) -> DataConditionResult:
-        if len(value.get("snuba_results", [])) != 1:
-            return False
-        return value["snuba_results"][0] > comparison["value"]
-
-
-class BaseEventFrequencyPercentHandler:
-    comparison_json_schema = {
-        "type": "object",
-        "properties": {
-            "interval": {"type": "string", "enum": list(STANDARD_INTERVALS.keys())},
-            "value": {"type": "integer", "minimum": 0},
-            "comparison_interval": {"type": "string", "enum": list(COMPARISON_INTERVALS.keys())},
-        },
-        "required": ["interval", "value", "comparison_interval"],
-        "additionalProperties": False,
-    }
-
-    @property
-    def intervals(self) -> dict[str, tuple[str, timedelta]]:
-        return PERCENT_INTERVALS
-
-    @staticmethod
-    def evaluate_value(value: WorkflowJob, comparison: Any) -> DataConditionResult:
-        if len(value.get("snuba_results", [])) != 2:
-            return False
-        return (
-            percent_increase(value["snuba_results"][0], value["snuba_results"][1])
-            > comparison["value"]
+slow_condition_query_handler_registry = Registry[type[BaseEventFrequencyQueryHandler]](
```

Comment:
using `type[BaseEventFrequencyQueryHandler]` ensures that the `batch_query` function is defined in all registered handlers, `BaseEventFrequencyQueryHandler` itself is not callable

Reply:
i was also getting all sorts of import errors trying to declare this in `workflow_engine/registry.py` because of importing `BaseEventFrequencyQueryHandler`, so i thought to put the registry here. i'm open to moving these handlers somewhere else since they're not exactly condition handlers to evaluate conditions like the other files in this folder

Reply:
👍 i think this is okay for now; i feel like the fix might be to break up `registry.py` into different files based on the types, then just have an __init__ file to fix any imports -- i can do that later if we end up wanting to.